### ISSUE TITLE
[bug] keybindings override system shortcuts

### ISSUE THREAD

**Issue by someone2037492034** [staniel359/muffon#211]
Weird thing now, probably a bug? On macOS, if I click play, then hide the window, go to another application/tab, and try to force quit the application (Say I'm in a browser, decide to force quit it with shortcut, muffon is hidden), instead of the application I'm in to quit, muffon quits, and it happens even if I don't have any windows open... It happens in version 2.1.0, version 2.0.3 doesn't have that issue.

**Comment by xyloflake** [comment]
@roloit can you attach a video of the bug? I can't reproduce it on my end (macos Monterrey).

**Comment by someone2037492034** [comment]
So first I start playing something, hide the window, then click the force quit shortcut while the application is hidden, it quits the application, it's very annoying when you try to quit other apps, and first, muffon quits, only then you can quit the application you're in.

**Comment by xyloflake** [comment]
> So first I start playing something, hide the window, then click the force quit shortcut while the application is hidden, it quits the application, it's very annoying when you try to quit other apps, and first, muffon quits, only then you can quit the application you're in. macOS Sequioa https://github.com/user-attachments/assets/e18b2e4e-8114-4e89-879a-b5efb13ce311

Hmm.... I'll check the code for just-in-case. 

For confirmation @roloit : you're holding down `Cmd+Q` right? Also please check the menu bar, on the top left, it should show the application name you're focussed on, if muffon is in focus, it will force quit.

**Comment by xyloflake** [comment]
Found the bug, no need to worry. Thank you for the report. I'll go ahead and report this.

@roloit this will be fixed in the next version. We apologize for the inconvenience caused.

https://github.com/staniel359/muffon/blob/b067d73825105452da8cfec700d1ba868f950b99/electron/actions/app/setupGlobalShortcuts.js#L16

@staniel359 @gamersi this registers a globalShortcut, meaning the shortcut will be registered system-wide. In this case `Cmd+Q` is registered system-wide as quit for muffon and even when the user uses the keybinding on another app they want to close, the keybinding fires the quit function. Solution would be to NOT register the shortcut globally but locally instead.

**Comment by someone2037492034** [comment]
I definitely was outside of muffon.
Thanks for finding the bug! I will probably use previous release right now.

**Comment by xyloflake** [comment]
> I definitely was outside of muffon. Thanks for finding the bug! I will probably use previous release right now.

Note: You don't have to wait for official releases. Our beta program offers facilities for testing the fixes (e.g., v2.1.0 had 23 beta tests). What you can do:
1. Wait for our fix.
2. Join the beta program.
3. Use the beta version until stability is confirmed.

TL;DR Wait for us to fix this issue, enroll in the beta program and use the beta until we declare the fix stable. I do not see an official release coming in the next 8 weeks. But betas are quite frequent.

**Comment by staniel359** [comment]
@roloit @xyloflake Got it, will fix.

Although it doesn't happen on Linux.
Don't know how I will test it.

**Comment by xyloflake** [comment]
@staniel359 can we fix this before v2.2.0? I believe this is pretty annoying.

**Comment by staniel359** [comment]
@xyloflake Sure.
Can you reproduce it on Mac?

**Comment by staniel359** [comment]
@roloit Will be fixed in the next version.

**Comment by gamersi** [comment]
(That's v2.2.0)
If you want to you can test it in the next beta will will release soon hopefully on our discord(just write me or Xylo or staniel and we will add you to the beta)

**Comment by someone2037492034** [comment]
Hello, no thanks, I won't enroll in the beta.

**Comment by staniel359** [comment]
@xyloflake It's fixed I guess?

**Comment by someone2037492034** [comment]
@staniel359, yes. I'll close the issue.
